Reminder-App/Reminder-App/Views/EditDateCell.swift

```diff
@@ -9,12 +9,28 @@ import UIKit
 
 class EditDateCell: UITableViewCell {
     
+    typealias DateChangeAction = (Date) -> Void
+    
     // MARK: Outlets
     @IBOutlet var datePicker: UIDatePicker!
     
+    // MARk: Private Properties
+    private var dateChangeAction: DateChangeAction?
+    
+    // MARK: Overridden Methods
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        datePicker.addTarget(self, action: #selector(dateChanged(_:)), for: .valueChanged)
+    }
+    
     // MARK: Public Methods
-    func configure(date: Date) {
-        
+    func configure(date: Date, changeAction: @escaping DateChangeAction) {
```

Comment:
DateChangeAction should be dateChangeAction

Reply:
`DateChangeAction` is a typealias i think. so its suppose to be like this.